go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/moby/sys/mountinfo v0.6.2
 	github.com/moby/sys/signal v0.7.0
 	github.com/moby/sys/symlink v0.2.0
+	github.com/montanaflynn/stats v0.6.6
```

Comment:
integration/client/ has its own go.mod. While the stats package is relatively small, it would be nice if we can have the package there instead of main go.mod.

https://github.com/containerd/containerd/tree/main/integration/client

Reply:
Good idea. @thaJeztah also suggested this in a parallel comment.

Reply:
Should integration/benchmark have its own go.mod or should it become part of the integration/client module?